partition/__main__.py

```diff
@@ -102,9 +98,9 @@ def main():
             print("Destination paths are all confirmed to be unique...")
 
         """ (6) Check for clobbering """
-        clobbered = clobbering(mapping)
-        if clobbered:
-            joined_clobbered = '\n'.join(clobbered)
+        existing_files = [filter(os.path.isfile, mapping.values())]
```

Comment:
This needs to call `list()` instead of using the `[...]` literal list syntax, otherwise you end up with a list with a single filter object (a type of generator) instead of the filtered values.